scheduled-jobs/build/bug-sweeps/4.5/Jenkinsfile

```diff
@@ -0,0 +1,28 @@
+properties(
```

Comment:
IIRC, @sosiouxme wants to get rid of scheduled sweeps, in favor of sweeping bugs after building something, so bugs and builds walk together ...